FlaskApp/api/user.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime 
+class User:
+    #class variable holds all users
+    users = []
```

Comment:
You have a class user for users I guess? but I cant see where your `__init__` is it makes me wonder how are you going to instantiate the user when you get the values of the user.

Reply:
Okey this is how i hundled the logic. The create user method returns a dictionary. Then directly appends the  user dictionary to the users list hence the list contains a list of dictionary containing user info.

Reply:
So with that logic how are you implementing the basics of `OOP`??